paper/paper.md

```diff
@@ -136,13 +136,25 @@ authentication) as well as processing functions for unpacking archives (zip or
 tar) and decompressing files (gzip, lzma, and bzip2).
 
 To the best of the authors' awareness, the only other Python software with some
-overlapping functionality is [Intake](https://github.com/intake/intake).
-While Intake is powerful and can be used to manage large data archives,
-we argue that Pooch has a simpler setup and meets the
-specific needs of scientific software authors and individual scientists.
-For example, Pooch does not require users to change their data loading code to
-fit into a plug-in structure, instead only providing the file path for the
-user.
+overlapping functionality are [Intake](https://github.com/intake/intake) and
+[fsspec](https://github.com/intake/filesystem_spec) (which is the backend for Intake).
```

Comment:
fsspec is also standalone and used by, for example, Dask (without Intake)

Reply:
PS: the `panel`-based GUI for selecting files from any backend currently lives in Intake, but should certainly be moved to fsspec.

Reply:
I'll clarify that it's not just a backend for Intake